src/views/MyProfile/Components/VictoryPromiseDisplay/index.js

```diff
@@ -371,19 +371,21 @@ export default class VictoryPromiseDisplay extends React.Component {
               <Grid container xs className="victory-promise-container-card-container-content">
                 {content}
               </Grid>
-              <Grid
-                container
-                justify="center"
-                className="victory-promise-container-card-container-link"
-              >
-                <Typography
-                  variant="body1"
-                  className="victory-promise-container-card-container-link-text"
-                  onClick={() => this.openVPLink()}
+              {this.props.network === 'online' && (
+                <Grid
+                  container
+                  justify="center"
+                  className="victory-promise-container-card-container-link"
                 >
-                  Click here for more information!
-                </Typography>
-              </Grid>
+                  <Typography
+                    variant="body1"
+                    className="victory-promise-container-card-container-link-text"
+                    onClick={() => this.openVPLink()}
+                  >
+                    Click here for more information!
+                  </Typography>
+                </Grid>
```

Comment:
If the user is offline on their My Profile page, the link that's displayed in the victory promise is removed since the user doesn't have an internet connection